CHANGELOG.md

```diff
@@ -136,6 +136,19 @@ If you're passing custom HTML into the cookie banner component (for example, usi
 
 This change was introduced in [pull request #2432: Remove default font styles from cookie banner Sass](https://github.com/alphagov/govuk-frontend/pull/2432).
 
+#### Import 'base' before importing Sass files from 'core' or 'overrides'
+
+If you import individual files from the `core` or `overrides` layers, you might see the error "Undefined mixin 'govuk-exports'" or "no mixin named govuk-exports" when compiling your Sass.
```

Comment:
```suggestion
If you import individual files from the `core` or `overrides` layers, you might see the error 'Undefined mixin 'govuk-exports'' or 'no mixin named govuk-exports' when compiling your Sass.
```
Reason: style guide advises single quotes.

Do users compile their Sass after they've imported individual files?

Reply:
@EoinShaughnessy I imagine Ollie did this because the double single quotes next to each other look a bit strange 🤔 Do you think we could wrap the error messages in back ticks, so they display like code? I think we'd normally do that for terminal commands/output?

Reply:
@vanitabarrett Great shout! Yeah, you're right, feels like terminal output should display like code.